airbyte-server/src/main/java/io/airbyte/server/apis/ConnectionApiController.java

```diff
@@ -31,9 +30,7 @@
 import io.micronaut.security.rules.SecurityRule;
 
 @Controller("/api/v1/connections")
-@Context()
-@Requires(property = "airbyte.deployment-mode",
-          value = "OSS")
+@Context
```

Comment:
Should we remove`@Context` or add it to the other controller as well? I'm not sure that it has any impact on cloud. It was just an annotation I added after noticing slowness on my local test.

Reply:
I think that we should do a pass on this after we get these changes in.  I'm hesitant to remove anything until we get everything working and have a baseline.